Framework/DataHandling/test/LoadEventNexusTest.h

```diff
@@ -914,26 +929,33 @@ class LoadEventNexusTest : public CxxTest::TestSuite {
 
 class LoadEventNexusTestPerformance : public CxxTest::TestSuite {
 public:
-#ifndef _WIN32
+#ifdef _WIN32
+  bool windows = true;
+#else
+  bool windows = false;
+#endif // _WIN32
```

Comment:
Maybe a solution to the problem that `ctest` does not support disabling tests: Put the tests in a different file, do not compile those tests on Windows. That is, use `cmake` to conditionally enable or disable them.

But you can also leave it as it is, I do not mind too much.